### ISSUE TITLE
Installation issue: astropy_helpers missing

### ISSUE THREAD

**Issue by barentsen** [sncosmo/sncosmo#180]
When trying to install sncosmo, I get:

```
$ pip install sncosmo
Collecting sncosmo
  Downloading sncosmo-1.4.0.tar.gz (789kB)
    100% |████████████████████████████████| 798kB 1.3MB/s 
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-m2y6b8os/sncosmo/setup.py", line 26, in <module>
        from astropy_helpers.setup_helpers import (
    ImportError: No module named 'astropy_helpers.setup_helpers'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-m2y6b8os/sncosmo/
```

**Comment by kbarbary** [comment]
Hmmm... it is supposed to automatically download astropy-helpers during install, if it isn't already present. I'll take a look.

In the meantime the conda install option should be more foolproof.

**Comment by kbarbary** [comment]
Couldn't reproduce on a clean conda environment. Do you happen to have an old `astropy_helpers` installed or something weird like that?

**Comment by barentsen** [comment]
I have astropy installed using `python setup.py develop`, which adds a symbolic link to the astropy source from `site-packages`.  This has the consequence of breaking `import astropy_helpers` because the astropy source dir contains astropy_helpers as a submodule directory.

TL;DR ignore this issue, it only affects weirdos who installed astropy using `python setup.py develop`.
